ci/Makefile

```diff
@@ -80,3 +80,9 @@ test_upgrade_apply_from_previous:
 .PHONY: test_upgrade_apply_user_lock
 test_upgrade_apply_user_lock:
 	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} test --verbose --suite test_skuba_upgrade.py --test test_upgrade_apply_user_lock
+
+.PHONY: test_pr_e2e
+test_pr:
+	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} test --verbose --suite test_cilium.py
+	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} test --verbose --suite test_deployments.py
+	${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} test --verbose --suite test_skuba_upgrade.py
```

Comment:
if this runs against every PR i would only run 1 upgrade test from the suite:

```
${TEST_RUNNER} -v vars.yaml -p ${PLATFORM} test --verbose --suite test_skuba_upgrade.py --test test_upgrade_plan_from_previous
```

Reply:
`test_cilium.py` is not 100% stable until https://github.com/SUSE/skuba/blob/master/ci/infra/testrunner/tests/test_cilium.py#L16 is replaced.